README.md

```diff
@@ -0,0 +1,35 @@
+This is the main development environment for working on Mozilla payments.
+
+You will need:
+* [Python](https://www.python.org/)
+* [Docker](https://docs.docker.com/)
+  * On Mac you can run this with
+    [VirtualBox](https://www.virtualbox.org/) and
+    [Boot2Docker](http://boot2docker.io/)
+* [docker-compose](https://docs.docker.com/compose/)
+
+Build your environment:
+* Check out the following repositories somewhere on your machine:
+  * [solitude](https://github.com/mozilla/solitude/)
+  * [payments-service](https://github.com/mozilla/payments-service/)
+* Run ``./link.py`` to connect your source.
+* Run ``docker-compose build`` to build the containers.
+
+Start up all containers:
+* Run ``docker-compose up -d``
+
+Launch the example site:
+* Run ``boot2docker ip`` to find your IP.
+* You may wish to add an entry in ``/etc/hosts`` for this
+  such as ``pay.dev``.
+* Open the example site to test out payments at
+  [http://pay.dev/](http://pay.dev/)
```

Comment:
My thought was that to test a payment you'd start at the example site, click buy, and then it would load the ``payments-ui`` site (in an iframe, I think).